packages/snaps-jest/src/matchers.test.tsx

```diff
@@ -217,25 +321,157 @@ describe('toSendNotification', () => {
           ],
         }),
       ).toSendNotification('foo', 'inApp'),
-    ).toThrow('Received:');
+    ).toThrow('Received');
   });
 
-  describe('not', () => {
-    it('passes when the notification is correct', () => {
+  it("fails when an expanded view notification's title is incorrect", () => {
+    const controllerMessenger = getRootControllerMessenger();
+    const actions = getInterfaceActions(
+      MOCK_SNAP_ID,
+      controllerMessenger as unknown as RootControllerMessenger,
+      {
+        content: (
+          <Box>
+            <Text>Foo</Text>
+          </Box>
+        ),
+        id: 'abcd',
+      },
+    );
+    expect(() =>
       expect(
         getMockResponse({
           notifications: [
             {
               id: '1',
-              type: 'native',
+              type: 'inApp',
               message: 'foo',
+              title: 'bar',
+              content: 'abcd',
             },
           ],
+          getInterface: () => {
+            return {
+              content: (
+                <Box>
+                  <Text>Foo</Text>
+                </Box>
+              ),
+              ...actions,
+            };
+          },
         }),
-      ).not.toSendNotification('bar');
-    });
+      ).toSendNotification(
+        'foo',
+        'inApp',
+        'baz',
+        <Box>
+          <Text>Foo</Text>
+        </Box>,
+      ),
+    ).toThrow('Received');
+  });
 
-    it('passes when the notification is correct with a type', () => {
+  it("fails when an expanded view notification's footerLink is incorrect", () => {
+    const controllerMessenger = getRootControllerMessenger();
+    const actions = getInterfaceActions(
+      MOCK_SNAP_ID,
+      controllerMessenger as unknown as RootControllerMessenger,
+      {
+        content: (
+          <Box>
+            <Text>Foo</Text>
+          </Box>
+        ),
+        id: 'abcd',
+      },
+    );
+    expect(() =>
+      expect(
```

Comment:
Why the double `expect`?

Reply:
Following the pattern other tests are using, you get "Property 'toThrow' does not exist on type 'void'." otherwise.